internal/controlplane/handlers_authz.go

```diff
@@ -530,32 +495,70 @@ func (s *Server) removeInvite(
 	role authz.Role,
 	email string,
 ) (*minder.RemoveRoleResponse, error) {
-	prj := targetPrj.String()
-	// Get all invitations for this email, project and role
-	inviteToRemove, err := s.store.GetInvitationByEmailAndProjectAndRole(ctx, db.GetInvitationByEmailAndProjectAndRoleParams{
+	// Get all invitations for this email and project
```

Comment:
A bit of a side-note, but I wonder if extracting all those non-exported methods like `removeInvite`, `removeRole` etc into interfaces (RoleService and InviteService maybe?) might make testing the area of code easier and would encapsulate the code better to only rely on what they need and not the whole big Server structure. Not for this PR though.